test/functional/postalcode/postalcode.js

```diff
@@ -0,0 +1,99 @@
+define([
+	"globalize",
+	"json!cldr-data/supplemental/likelySubtags.json",
+	"json!cldr-data/supplemental/postalCodeData.json",
+	"../../util",
+	"json!./dummy/kr.json",
+	"json!./dummy/us.json",
+
+	"globalize/postalcode"
+
+], function( Globalize, likelySubtags, postalcode, util, kr, us ) {
+
+	QUnit.module( ".validatePostalCode( value )", {
+		setup: function() {
+			Globalize.load( likelySubtags, {
+				main: {
+					ko: {}
+				}
+			});
+			Globalize.locale( "ko" );
```

Comment:
Well observed hack (using empty main ko data to allow ko bundles given we aren't actually using any ko data for postalcode manipulation). Although, the more I see how we are using postalcode methods, the more I'm convinced we should require the `region` argument and avoid using cldr's region.